lib/ui.rb

```diff
@@ -2,20 +2,6 @@
 
 class UserInterface
 
-    def display_board(board)
-        line_break = "\n"
-        grid_lines = "===+===+==="
-        board_grid = line_break +
-        " #{board[0]} | #{board[1]} | #{board[2]} " + line_break +
-        grid_lines + line_break +
-        " #{board[3]} | #{board[4]} | #{board[5]} " + line_break +
-        grid_lines + line_break +
-        " #{board[6]} | #{board[7]} | #{board[8]} " + line_break +
-        line_break
-        
-        return board_grid
-    end
-
     def display(message)
```

Comment:
Nice! I still think you can include a `display_board` method here that takes in a (formatted) board parameter and prints it. Different than printing a message (which takes a message parameter). So two separate methods here. Make sense?

Reply:
Then maybe you change your `display` method to `display_message` method so you know the distinction between the two.

Reply:
Just updated this per your suggestion and added another test for the new `display_board` `UserInterface` method!